sdk/identity/azure_identity/src/app_service_managed_identity_credential.rs

```diff
@@ -14,7 +14,7 @@ const SECRET_HEADER: HeaderName = HeaderName::from_static("x-identity-header");
 const SECRET_ENV: &str = "IDENTITY_HEADER";
 
 #[derive(Debug)]
-pub struct AppServiceManagedIdentityCredential {
+pub(crate) struct AppServiceManagedIdentityCredential {
```

Comment:
Tip: why `pub(crate)`? Just to be extra safe? As long as it doesn't get exposed publicly from `src/lib.rs` it can be just `pub`.